src/components/Notification/NotifDrawer/NotifDrawer.js

```diff
@@ -0,0 +1,34 @@
+import ClassNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const NotifDrawer = ({ hide, expanded, children, className, ...props }) => {
+  const classes = ClassNames(
+    'drawer-pf drawer-pf-notifications-non-clickable',
+    { 'drawer-pf-expanded': expanded },
```

Comment:
@ohadlevy

Reply:
wouldnt this just decide if its expanded or not? i was referring to not having the expand option, if you dont have a lot of content.

Reply:
You want the expand button to be optional? on the way..